### DISCUSSION TITLE
I don't know how to define job run time

### DISCUSSION THREAD

**Discussion by breezechime** [agronholm/apscheduler#762] [Q&A] [answered]
### Things to check first

- [X] I have searched the existing issues and didn't find my feature already requested there

### Feature description

The following code cannot execute properly, It seems that nesting is not supported.
I hope my tasks are scheduled from 9:28 to 11:30 and from 13:00 to 15:00 on weekdays.
Who can tell me how to define it this way? I don't want to define multiple tasks
expample: 
```python
trigger = AndTrigger([
            IntervalTrigger(seconds=11),
            OrTrigger([
                CronTrigger(day_of_week='0-4'),
                CronTrigger(day_of_week='0-4', hour='10', minute='0-59'),
                CronTrigger(day_of_week='0-4', hour='11', minute='0-30'),
                CronTrigger(day_of_week='0-4', hour='13-14', minute='0-59'),
                CronTrigger(day_of_week='0-4', hour='15', minute='0-1'),
            ])
        ])
        # A股
        self.scheduler.add_job(func=self._callJob, args=(self.queue_spider, fetchAMarketSnapshot),
                               trigger=trigger, max_instances=100)
```

### Use case

This is very important. If a more complex homework time cannot be defined, it will only be applied to simple areas

**Comment by agronholm** [comment]
This should be a post on the Q/A section of the discussion forum, not a feature request.

**Reply by agronholm** [reply]
Is it correct that you want your task to be run on 11 second intervals within the bounds you described? IntervalTrigger and CronTrigger don't mix well. Perhaps you should use `seconds='*/10'` as a trigger? It's not 11 but close.

**Reply by breezechime** [reply]
Yes, I would like the task to be repeated within the description range with an interval of 11 seconds

**Reply by breezechime** [reply]
Can you tell me a better solution for this example

**Reply by agronholm** [reply, marked as answer]
Assuming a 10 second interval works too:
```python
trigger = OrTrigger([
    CronTrigger(day_of_week='0-4', hour='9', minute='28-59', second='*/10'),
    CronTrigger(day_of_week='0-4', hour='10', second='*/10'),
    CronTrigger(day_of_week='0-4', hour='11', minute='0-30', second='*/10'),
    CronTrigger(day_of_week='0-4', hour='13-14', second='*/10')),
    CronTrigger(day_of_week='0-4', hour='15'),
])
```
